src/main/java/com/puppycrawl/tools/checkstyle/TreeWalker.java

```diff
@@ -388,8 +388,10 @@ public Set<String> getExternalResourceLocations() {
         return Stream.concat(filters.stream(),
                 Stream.concat(ordinaryChecks.stream(), commentChecks.stream()))
             .filter(ExternalResourceHolder.class::isInstance)
-            .map(ExternalResourceHolder.class::cast)
```

Comment:
@Kevin222004 I am curious; does this suffer the same surviving mutation?

```
            .map(r -> (ExternalResourceHolder) r)
```

Reply:
@nrmancuso Yes, it will survive the same.

https://kevin222004.github.io/PitestReport/com.puppycrawl.tools.checkstyle/TreeWalker.java.html